climt/_components/rrtmg/lw/component.py

```diff
@@ -84,113 +109,63 @@ class RRTMGLongwave(ClimtPrognostic):
 
     def __init__(
             self,
-            calculate_change_up_flux=0,
-            cloud_overlap_method=1,
-            cloud_optical_properties=2,
-            cloud_ice_properties=1,
-            cloud_liquid_water_properties=1,
-            acceleration_gravity=None,
-            planck_constant=None,
-            boltzmann_constant=None,
-            speed_of_light=None,
-            avogadro_constant=None,
-            loschmidt_constant=None,
-            universal_gas_constant=None,
-            stefan_boltzmann_constant=None,
-            seconds_per_day=None,
-            specific_heat_dry_air=None):
+            calculate_change_up_flux=False,
+            cloud_overlap_method='random',
+            cloud_optical_properties='liquid_and_ice_clouds',
+            cloud_ice_properties='ebert_curry_two',
+            cloud_liquid_water_properties='radius_dependent_absorption',
+            calculate_interface_temperature=True):
 
         """
 
         Args:
 
-            calculate_change_up_flux (int):
+            calculate_change_up_flux (bool):
                 calculate derivative of flux change with respect to
                 surface temperature alone. Can be used to adjust fluxes in between radiation calls
-                only due to change of surface temperature. Default value is 0, meaning this quantity
+                only due to change of surface temperature. Default value is :code:`False`, meaning this quantity
                 is not calculated.
 
-            cloud_overlap_method (int):
+            cloud_overlap_method (string):
                 Choose the method to do overlap with:
 
-                * 0 = Clear only (no clouds)
-                * 1 = Random
-                * 2 = Maximum/Random
-                * 3 = Maximum.
-
-                Default value is 1, corresponding to random cloud overlap.
+                * :code:`clear_only` = Clear only (no clouds)
+                * :code:`random` = Random
+                * :code:`maximum_random` = Maximum/Random
```

Comment:
maximum overlap of adjacent but random overlap of separate cloudy layers: 
http://journals.ametsoc.org/doi/full/10.1175/1520-0493(1998)126%3C3336%3AELCFAM%3E2.0.CO%3B2

some parameterization to calculate absorptivity of clouds, since a given grid cell need not be fully cloudy...